utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -407,6 +407,10 @@ int main(int argc, char **argv)
     if(testCase == 46)
         CHECK_RETURN_STATUS(hipHostMalloc(&intensity, batchSize * sizeof(Rpp32f)));
 
+    Rpp32f *gamma;
```

Comment:
why the color shown is different?
Is there any predefined gamma function present anywhere?

Reply:
gamma is predefined function in c++. I changed to gammaTensor .